### PR TITLE
Added a TPB to catch Unit 42 breaking changes

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
After some breaking changes in Unit 42, added a TPB to catch those changes before the customers.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py [hunk 1]
```diff
@@ -662,13 +662,19 @@ def get_indicators_command(client: Client, args: Dict[str, str], feed_tags: Opti
     Returns:
         Demisto Outputs.
     """
+    is_up_to_6_2 = is_demisto_version_ge('6.2.0')
```
Comment (guykeller) [inline Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py:665]: It isn't up to 6.2
```suggestion
    is_version_over_6_2= is_demisto_version_ge('6.2.0')
```

File: Packs/FeedUnit42v2/ReleaseNotes/1_0_9.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Unit 42 ATOMs Feed
+- Maintenance and stability enhancements.
```
Comment (guykeller) [inline Packs/FeedUnit42v2/ReleaseNotes/1_0_9.md:4]: Why the choice of no actual RN here?

File: Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py (hunk 1 above)
Comment (ChanochShayner) [reply Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py]: done

File: Packs/FeedUnit42v2/ReleaseNotes/1_0_9.md (hunk 2 above)
Comment (ChanochShayner) [reply Packs/FeedUnit42v2/ReleaseNotes/1_0_9.md]: done

**Review by guykeller — APPROVED** [review]

**Pull request merged**